docs/plans/plan-master-a-must-do.md

```
@@ -0,0 +1,160 @@
+|-----|-------|----------|
+| Add `cd shadow-agent` to npm commands | `plan-finish-line.md` | Low — accuracy |
+| Fix markdown ordered-list numbering | `plan-finish-line.md:74` | Low — lint |
+
+### PRs #27, #28, #30 — No blocking inline comments found
+These have REVIEW_REQUIRED status. No actionable code fixes needed — just need review approval.
+
+---
+
+## 2) Merge Sequence (dependency-safe)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)